SPECS-EXTENDED/kernel-lvbs/kernel-lvbs.spec

```diff
@@ -0,0 +1,1925 @@
+#  10. Verify the rest of the config file looks ok
+# If there are significant changes to the config file, disable the config check and build the
+# kernel rpm. The final config file is included in /boot in the rpm.
+
+# LVBS is x86_64 only, for now
+ExclusiveArch: x86_64
+
+%description
+The kernel package contains the Linux kernel.
+
+%package devel
+Summary:        Kernel Dev
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+Requires:       gawk
+Requires:       python3
+Obsoletes:      linux-dev
+
+%description devel
+This package contains the Linux kernel dev files
+
+%package drivers-accessibility
+Summary:        Kernel accessibility modules
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+
+%description drivers-accessibility
+This package contains the Linux kernel accessibility support
+
+%package drivers-gpu
+Summary:        Kernel gpu modules
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+
+%description drivers-gpu
+This package contains the Linux kernel gpu support
+
+%package drivers-intree-amdgpu
+Summary:        Kernel amdgpu modules
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+Requires:       %{name}-drivers-gpu = %{version}-%{release}
+
+%description drivers-intree-amdgpu
+This package contains the Linux kernel in-tree AMD gpu support
+
+%package drivers-sound
+Summary:        Kernel Sound modules
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+
+%description drivers-sound
+This package contains the Linux kernel sound support
+
+%package docs
+Summary:        Kernel docs
+Group:          System Environment/Kernel
+Requires:       python3
+
+%description docs
+This package contains the Linux kernel doc files
+
+%package tools
+Summary:        This package contains the 'perf' performance analysis tools for Linux kernel
+Group:          System/Tools
+Requires:       %{name} = %{version}-%{release}
+Requires:       audit
+
+%description tools
+This package contains the 'perf' performance analysis tools for Linux kernel.
+
+%package -n     python3-perf
+Summary:        Python 3 extension for perf tools
+Requires:       python3
+
+%description -n python3-perf
+This package contains the Python 3 extension for the 'perf' performance analysis tools for Linux kernel.
+
+%package -n     bpftool
+Summary:        Inspection and simple manipulation of eBPF programs and maps
+
+%description -n bpftool
+This package contains the bpftool, which allows inspection and simple
+manipulation of eBPF programs and maps.
+
+%prep
+%autosetup -p1 -n CBL-Mariner-Linux-Kernel-rolling-lts-mariner-%{mariner_version}-%{version}
+
+CheckConfig() {
+    local CONFIG_FILE=$1
+
+    export KBUILD_OUTPUT=$2
+
+    make mrproper
+
+    cp $CONFIG_FILE $KBUILD_OUTPUT/.config
```

Comment:
Just checking but will this work since the `lvbs_normal` and `lvbs_secure` directories have not been created yet?

Reply:
yeah, `make mrproper` will create the `$KBUILD_OUTPUT` dir